internal/controller/promotions/promotions_test.go

```diff
@@ -569,14 +569,14 @@ func Test_parseCreateActorAnnotation(t *testing.T) {
 func Test_calculateRequeueInterval(t *testing.T) {
 	testStepKindWithoutTimeout := "fake-step-without-timeout"
 	promotion.RegisterStepRunner(
-		&pkgPromotion.MockStepRunner{Nm: testStepKindWithoutTimeout},
+		&pkgPromotion.MockStepRunner{StepName: testStepKindWithoutTimeout},
```

Comment:
That's nicer than `Nm` I was tired when I did that and grumpy that `Name` conflicted with `Name()`. Thank you.